ui/admin/setup-edit.php

```diff
@@ -134,6 +134,17 @@
     if ( empty( $field_type_fields ) )
         $no_additional[] = $field_type;
 }
+
+/**
+ * Make use of the WP core meta box functionality
+ * 
+ * Currently only context 'side' is available
+ * 
+ * @since 2.7
+ * @see https://codex.wordpress.org/Plugin_API/Action_Reference/add_meta_boxes
+ * @param array $pod The Pod object as an array
+ */
+do_action( 'add_meta_boxes', 'pods_edit', $pod );
```

Comment:
Because in one of the next feature releases we'll have groups editor AND fields editor (within a group). Let's define that now as edit-fields, then the other can be edit-groups, and we can continue etc